package.json

```diff
@@ -11,7 +11,9 @@
     "build": "ember build",
     "start": "ember server",
     "test": "npm run lint && ember test",
-    "lint": "eslint *.js addon app blueprints config tests --fix"
+    "eslint": "eslint *.js addon app blueprints config tests --fix",
+    "sass-lint": "sass-lint -q -v",
+    "lint": "npm run eslint && npm run sass-lint"
```

Comment:
I personally prefer to make these script names more generic like `lint-js` and `lint-css` in case we ever switch from ESLint to something else or from SASS to something else (such as Less).